.storybook/main.js

```diff
@@ -53,7 +53,7 @@ module.exports = {
           $journey: resolve('./src/lib/journey'),
           $lib: resolve('./src/lib'),
           $locales: resolve('./src/locales'),
-          $widget: resolve('./src/lib/widget'),
+          $widget: resolve('./src/widget'),
```

Comment:
Moved the `widget` directory out of the `lib` directory as I'm considering anything in `lib` to be a reusable component, but the top-level widget stuff is the entry file into the app, which isn't a reusable thing ... kind of like the routes.